docs/components/output-mode-table.mdx

```diff
@@ -1,6 +1,7 @@
-| option    | description                              |
-| --------- | ---------------------------------------- |
-| full      | This is the default. Displays all output |
-| hash-only | Show only the hashes of the tasks        |
-| new-only  | Only show output from cache misses       |
-| none      | Hides all task output                    |
+| option      | description                              |
+| ----------- | ---------------------------------------- |
+| full        | This is the default. Displays all output |
+| hash-only   | Show only the hashes of the tasks        |
+| new-only    | Only show output from cache misses       |
+| errors-only | Only show output from task failures      |
+| none        | Hides all task output                    |
```

Comment:
Can you move docs to a separate PR? Since the feature won't be out until the next release, we'll hold onto the docs PR until then.